hack/packages/package-tools/cmd/package-bundle-generate.go

```diff
@@ -30,7 +30,6 @@ var packageBundleGenerateCmd = &cobra.Command{
 
 func init() {
 	packageBundleCmd.AddCommand(packageBundleGenerateCmd)
-	packageBundleGenerateCmd.Flags().StringVar(&packageRepository, "repository", "", "Package repository of the package bundle being created")
```

Comment:
i would still like to keep `packageRepository`, but make it optional.. that way if someone is working just on that package repository they can build and play with packages in that repo

Reply:
Good call. I have made PACKAGE_REPOSITORY an option now.